auth-web/src/models/affiliation.ts

```diff
@@ -43,14 +43,37 @@ export interface AffiliationResponse {
   lastModified?: string
   modified?: string
   modifiedBy?: string
-  nameRequest?: NameRequest
+  nameRequest?: NameRequestResponse
   nrNumber?: string
   status?: string
   goodStanding?: boolean
   adminFreeze?: boolean
   dissolved?: boolean
 }
 
+export interface NameRequestResponse {
+  actions?: Array<Action>
+  consentFlag?: string
+  names?: Array<Names>
+  id?: number
+  legalType: CorpTypes
+  state?: string
+  applicantEmail?: string
+  applicantPhone?: string
+  enableIncorporation?: boolean
+  folioNumber?: string
+  target?: NrTargetTypes
+  entityTypeCd?: string
+  natureOfBusiness?: string
+  expirationDate?: Date
+  nrNum?: string
+  stateCd?: string
```

Comment:
I see `state` and then `stateCd`?

Reply:
yeah `stateCd` under nameRequest from api response